.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
```

Comment:
Does OpenShift use these labels?

Reply:
Resolved in https://github.com/operator-framework/operator-lifecycle-manager/pull/1319#discussion_r383455764